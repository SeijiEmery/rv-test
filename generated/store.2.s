addi t1, zero, 200
    sd   a0, 0(t1)
    sw   a0, 8(t1)
    sh   a0, 16(t1)
    sb   a0, 24(t1)

    ld   a1, 0(t1)
    ld   a2, 8(t1)
    ld   a3, 16(t1)
    ld   a4, 24(t1)
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
ebreak
