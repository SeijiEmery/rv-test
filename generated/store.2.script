load /x 0 generated/store.2.hex
writereg 10 4099
setpc 0
run 1004
readreg 10
readreg 11
readreg 12
readreg 13
readreg 14