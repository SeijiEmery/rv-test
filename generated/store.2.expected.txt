tests/store.test.s:44
R10 = 4099
R11 = 4099
R12 = 4099
R13 = 4099
R14 = 3
